applications/FluidDynamicsBiomedicalApplication/tests/test_FluidDynamicsBiomedicalApplication.py

```diff
@@ -48,7 +48,7 @@ def AssembleTestSuites():
     KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(KratosMultiphysics.Logger.Severity.WARNING)
     KratosMultiphysics.Logger.PrintInfo("Unittests", "\nRunning cpp unit tests ...")
     KratosMultiphysics.Tester.SetVerbosity(KratosMultiphysics.Tester.Verbosity.PROGRESS) # TESTS_OUTPUTS
-    KratosMultiphysics.Tester.RunTestSuite("FluidDynamicsBiomedicalApplicationFastSuite")
+    # KratosMultiphysics.Tester.RunTestSuite("FluidDynamicsBiomedicalApplicationFastSuite")
```

Comment:
```suggestion
    KratosMultiphysics.Tester.RunTestSuite("FluidDynamicsBiomedicalApplicationFastSuite")
```
I think this is another leftover.

Reply:
Mm I am not sure, this assumes that there is a existing test suite with that name, and if its not the case it crashes (for example because the cpp tests are not compiled). If we want to leave this enabled we have to control when is executed.

Reply:
But the CI always compiles the cpp tests right?

Reply:
But its different, the CI runs the cpp tests though the runner scrips. You don't specify which test to run, you check the ones that are available and you run them all. With this you are explicitly asking to run the tests in a suit that may not exists.

It should be at least warded with a try/catch

Reply:
Understood. Lets keep it commented then.